pkg/infra/process/root_check.go

```diff
@@ -1,3 +1,4 @@
+//go:build !windows
```

Comment:
is this required?  can we remove this the infra/* changes?

Reply:
done